deno/src/platform.ts

```diff
@@ -46,7 +48,11 @@ export const inputFileData = Symbol('InputFile data')
  * Reference](https://core.telegram.org/bots/api#inputfile).
  */
 export class InputFile {
-    public readonly [inputFileData]: ConstructorParameters<typeof InputFile>[0]
+    private [internalInputFileData]: ConstructorParameters<typeof InputFile>[0]
+    public get [inputFileData]() {
+        const file = this[internalInputFileData]
+        return file instanceof Blob ? file.stream() : file
+    }
```

Comment:
I know what I dislike about this change. This logic should be moved into `Client`.

Reply:
InputFile is semantically part of the client. The two cannot be separated. It just has to reside in a different file for cross-platform reasons.

In a way, the InputFile encapsulates the runtime-specific part of the client. Given that calling `stream()` only works on Deno, it needs to be part of InputFile.

That being said, I know this way to divide the code is mediocre at best. Maybe d2n will improve the situation.

Reply:
d2n should be able to shim `Blob`, so that it's possible to `if (Blob && foo instanceof Blob)`.

Reply:
Node 16.7.0 added `Blob::stream()`